Assets/UI/WorldView/plotinfo.lua

```diff
@@ -937,15 +1155,6 @@ function OnPlayerTurnActivated( ePlayer:number, isFirstTimeThisTurn:boolean )
   end
 end
 
-
-function KeyHandler( key:number )
-  if key == Keys.VK_TAB then
-    UILens.ClearLayerHexes( LensLayers.MAP_HEX_MASK );   -- ??TRON debug clear
-    return true;
-  end
-  return false;
-end
-
 function OnInputHandler( pInputStruct:table )
```

Comment:
I assume you left this function in because it was modified by CQUI, but you should remove it and just make a note in your PR mentioning that this functionality is now missing. I looked through the rest of the diff and don't really see where this would go now, but @Azurency might know.

This is just my opinion though, might want to wait with this until @Azurency comments.

If you do remove this function, make sure you remove the reference to it on [line 1213](https://github.com/Azurency/CQUI_Community-Edition/pull/384/files#diff-80f9bce53b0ae8a96e3056ffed4fc5c6R1213) as well (can't  comment on that line since it wasn't modified, I think).

Reply:
Put KeyHandler back in so now I'm syntactically correct, will wait for further guidance.